### PR TITLE
feat(Sketch.js): allow fromFile api to receive a file as buffer OR path

### PR DESCRIPTION
**Pull request by MichaelZaporozhets**
It is very inconvenient when integrating multiple systems working with sketch 'files' to need to
write to the filesystem each time they would like to use sketch-constructor. Adding the ability for
the file to be either a buffer or path solves this issue cleanly, without needing to change the api.

*Description of changes:*
- added logic to allow function parameter to be either Buffer or file path String

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: models/Sketch/Sketch.test.js [hunk 1]
```diff
@@ -35,6 +35,14 @@ describe('Sketch', () => {
       });
     });
 
+    it('should work when file is Buffer', () => {
+      expect(true).toBeTruthy();
```
Comment (jorgecasar) [inline models/Sketch/Sketch.test.js]: This expect is not needed

File: models/Sketch/Sketch.test.js [hunk 2]
```diff
@@ -35,6 +35,14 @@ describe('Sketch', () => {
...
+      const fileBuffer = fs.readFileSync(`${process.cwd()}/__tests__/__sketch-files/test.sketch`);
+      return Sketch.fromFile(fileBuffer).then(sketch => {
+        expect(sketch).toBeDefined();
```
Comment (jorgecasar) [inline models/Sketch/Sketch.test.js]: It's needed to manage the asynchrony. Use `done` callback or `async/await`.

File: models/Sketch/Sketch.test.js (hunk 1 above)
Comment (MichaelZaporozhets) [reply models/Sketch/Sketch.test.js]: ah- this one was present previously in the test above. I've removed it from that one too.

File: models/Sketch/Sketch.test.js (hunk 2 above)
Comment (MichaelZaporozhets) [reply models/Sketch/Sketch.test.js]: good call. Updated.

**Comment by MichaelZaporozhets** [comment]
> The code looks good, did you mean to delete the package-lock and include yarn.lock? If we want to switch from npm to yarn, we should discuss that in another issue and PR. Thanks!

Danny, sorry about that- Travis was complaining about the presence (previously) of the package lock, or at least I thought so. Turns out I had it backwards. I'll de-yarnify- although a switch to yarn might be nice!

**Comment by MichaelZaporozhets** [comment]
success! Thank you @dbanksdesign & @jorgecasar

**Pull request merged**